js/geo/coordinate.js

```diff
@@ -15,6 +15,7 @@ function Coordinate(column, row, zoom) {
     this.column = column;
     this.row = row;
     this.zoom = zoom;
+    this.posMatrix = null;
```

Comment:
I think precalculating the matrix and storing it on TileCoord is fine. TileCoord is the closest thing to -native's `Tile` that we have in -js now.